Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -535,12 +532,4 @@ open class ChatMessageListVC:
         // To prevent the gesture recognizer consuming up the events from UIControls, we receive touch only when the view isn't a UIControl.
         !(touch.view is UIControl)
     }
-
-    // MARK: - UIAdaptivePresentationControllerDelegate
-
-    public func presentationControllerShouldDismiss(_ presentationController: UIPresentationController) -> Bool {
```

Comment:
We're removing public API here. It would be nice to mention the reasoning behind this decision ;)

Reply:
I thought about deprecating it, but I don't think anyone is using this anyway, and you could not override it (since it is public) so it is safe to remove this one. Or do you mean comment in the commit?

Reply:
✅